helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -13,5 +13,23 @@ data:
         region: '{{ .Values.Installation.V1.Provider.AWS.Region }}'
       installation:
         name: '{{ .Values.Installation.V1.Name }}'
+        {{- if .Values.Installation.V1.Guest }}
+        guest:
+          {{- if .Values.Installation.V1.Guest.Kubernetes }}
```

Comment:
Can we skip the ifs, and handle empty values in the code instead? We have default values in installations anyway which can be simply empty strings.

Reply:
There was two possible ways:
  1. Add default values for all envs in installation - in this case, we will not need those ifs
  2. Do not add default values and check nested keys in configmap template. In this case, those checks should be done on every layer (best practices from here https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md#flat-or-nested-values)

I took the first approach. But I'm also okay to go with second

Reply:
I'd rather add that to default values. Then it's easier to see what settings we have globally.